clients/java/src/main/java/io/camunda/zeebe/client/api/search/filter/UserTaskVariableFilter.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.search.filter;
+
+import io.camunda.zeebe.client.api.search.query.TypedSearchQueryRequest.SearchRequestFilter;
+
+public interface UserTaskVariableFilter extends SearchRequestFilter {}
```

Comment:
🤔 can we reuse `VariableFilter` here ?

Reply:
We cant for one reason, as we are using on the service the Variable Filter over the scopeKeys, add the Variable filter here would override the VariableFilter applied on the service filter. 

We can in the future  add filter capabilities over here, but so far, I dont see user case, once if they wanna navigate over the variables in another level of detail, it can use the Search Variables.